src/tables/extensions/databaseTable.tsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import useSWR from "swr";
+import { openmrsFetch } from "@openmrs/esm-framework";
+
+import { InfoDataTable } from "./dataTable";
+
+const DatabaseTable: React.FC = () => {
+  const url = `/ws/rest/v1/systeminformation`;
+  const { data, error, isValidating } = useSWR<
+    { data: { systemInformation } },
+    Error
+  >(url, openmrsFetch);
+
+  const isLoading = !data && !error;
+
+  const headerData = [
```

Comment:
Static variables with no dependence on the component can be defined outside of the component to improve render cycles

Reply:
The data is going to be different, I created a reusable component for the tables that will receive rows and headers as props, these five tables will have different headers and rowData.